generators/app/templates/src/app/screens/Dashboard/screens/Home/index.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+
+import withProvider from '~components/ProviderWrapper';
 
 import logo from './assets/logo.svg';
 import styles from './styles.module.scss';
+import { useSelector, Context, useDispatch } from './context';
+import { reducer, INITIAL_STATE, actionCreators } from './reducer';
 
 function Home() {
+  const foo = useSelector(state => state.foo);
```

Comment:
I'd add a comment to let a new user know why this is here. Otherwise it might seem a little confusing

Reply:
Sure, nevertheless this screen will always be deleted, it's the default "Learn React" screen.